analyses/focal-cn-file-preparation/run-prepare-cn.sh

```diff
@@ -19,92 +19,80 @@ script_directory="$(perl -e 'use File::Basename;
-      --metadata $histologies_file \
-      --goi_list $goi_file \
-      --filename_lead "consensus_seg_annotated_cn"_${chromosome_type}_${strategy}
-  done
-done
+--cnv_file ${scratch_dir}/consensus_seg_with_status.tsv \
+--gtf_file $gtf_file \
+--metadata $histologies_file \
+--filename_lead "consensus_seg_annotated_cn" \
+--seg
+
 
 # if we want to process the CNV data from the original callers
 # (e.g., CNVkit, ControlFreeC)
 if [ "$RUN_ORIGINAL" -gt "0" ]; then
 
-  # Prep the CNVkit data
-  Rscript --vanilla -e "rmarkdown::render('01-add-ploidy-cnvkit.Rmd', clean = TRUE)"
-
-  # Run annotation step for CNVkit
-  Rscript --vanilla 04-prepare-cn-file.R \
-    --cnv_file ${scratch_dir}/cnvkit_with_status.tsv \
-    --gtf_file $gtf_file \
-    --metadata $histologies_file \
-    --filename_lead "cnvkit_annotated_cn" \
-    --seg
-
-  # Run annotation step for ControlFreeC
-  Rscript --vanilla 04-prepare-cn-file.R \
-    --cnv_file ${data_dir}/cnv-controlfreec.tsv.gz \
-    --gtf_file $gtf_file \
-    --metadata $histologies_file \
-    --filename_lead "controlfreec_annotated_cn" \
-    --controlfreec
-
-  filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn")
-  for filename in ${filenameLead[@]}; do
-    for strategy in ${libraryStrategies[@]}; do
-      for chromosome_type in ${chromosomesType[@]}; do
-        Rscript --vanilla rna-expression-validation.R \
-          --annotated_cnv_file results/${filename}_${chromosome_type}.tsv.gz \
-          --expression_file ${data_dir}/gene-expression-rsem-fpkm-collapsed.${strategy}.rds \
-          --independent_specimens_file $independent_specimens_file \
-          --metadata $histologies_file \
-          --goi_list $goi_file \
-          --filename_lead ${filename}_${chromosome_type}_${strategy}
-      done
-    done
-  done
+# Prep the CNVkit data
+Rscript --vanilla -e "rmarkdown::render('01-add-ploidy-cnvkit.Rmd', clean = TRUE)"
 
+# Run annotation step for CNVkit
+Rscript --vanilla 04-prepare-cn-file.R \
+--cnv_file ${scratch_dir}/cnvkit_with_status.tsv \
+--gtf_file $gtf_file \
+--metadata $histologies_file \
+--filename_lead "cnvkit_annotated_cn" \
+--seg \
+--runWXSonly
+
+# Run annotation step for ControlFreeC
+Rscript --vanilla 04-prepare-cn-file.R \
+--cnv_file ${data_dir}/cnv-controlfreec.tsv.gz \
+--gtf_file $gtf_file \
+--metadata $histologies_file \
+--filename_lead "controlfreec_annotated_cn" \
+--controlfreec \
+--runWXSonly
+
+# Run merging for all annotated files 
+Rscript --vanilla 07-consensus-annotated-merge.R \
+--cnvkit_auto ${results_dir}/cnvkit_annotated_cn_wxs_autosomes.tsv.gz \
+--cnvkit_x_and_y ${results_dir}/cnvkit_annotated_cn_wxs_x_and_y.tsv.gz \
+--consensus_auto ${results_dir}/consensus_seg_annotated_cn_autosomes.tsv.gz \
+--consensus_x_and_y ${results_dir}/consensus_seg_annotated_cn_x_and_y.tsv.gz \
+--outdir ${results_dir}
+
+
+# filenameLead=("cnvkit_annotated_cn" "controlfreec_annotated_cn" "cnvkit_annotated_cn_wxs" "controlfreec_annotated_cn_wxs")
```

Comment:
Should we also remove these commented-out lines since it would not be needed at this moment?

Reply:
@runjin326 Sorry I missed removing them. May be I can remove them in the next PR while also removing the download lines for gencode v38?